knimin/templates/pm_targeted_pool.html

```diff
@@ -0,0 +1,324 @@
+      var numSamples = $(".plate-select option[value=" + $(this).val() + "]").data('pm-samples');
+      $(this).parents('.entry:first').find('.pm-num-samples:first').val(numSamples);
+      $(this).parents('.entry:first').find('.pm-percentage:first').trigger('change');
+    });
+
+    // Update the computed values
+    updateTotalValues();
+  }
+
+  $(document).ready(function () {
+    // Modofy the button of the last plate entry to have a + sign instead of -
+    $('.entry:first .btn-remove')
+      .removeClass('btn-remove').addClass('btn-add')
+      .removeClass('btn-info').addClass('btn-success')
+      .html($('<span>').addClass('glyphicon glyphicon-plus'));
+
+    $('.glyphicon-warning-sign').hide();
+
+    $('#run-pool-volume-input').change(function() {
+      runTotalPoolVolumeUpdated();
+    });
+
+    initGUI();
+
+    // Dynamically add/remove plate list
+    $(document).on('click', '.btn-add', function(e) {
+      e.preventDefault();
+
+      var controlForm = $('.tag-controls');
+      var currentEntry = $(this).parents('.entry:first');
+      var newEntry = $('<div class="entry row">' +
+                         '<div class="col-xs-5 form-group">' +
+                           '<select class="form-control plate-select">' +
+                             '<option value="-1" selected disabled>Choose a dna-plate</option>' +
+                           '</select>' +
+                         '</div>' +
+                         '<div class="col-xs-2 form-group">' +
+                           '<input class="form-control pm-volume" type="number" value="240"/>' +
+                         '</div>' +
+                         '<div class="col-xs-1 form-group">' +
+                           '<input class="form-control pm-num-samples" type="number" min="0" value="0" disabled/>' +
+                         '</div>' +
+                         '<div class="col-xs-1 form-group">' +
+                           '<input class="form-control pm-percentage" type="number" min="0" max=100 value="10" step="0.1"/>' +
+                         '</div>' +
+                         '<div class="col-xs-2 form-group">' +
+                           '<input class="form-control pm-plate-pool" type="number" min="0" value="0" step="0.1"/>' +
+                         '</div>' +
+                         '<div class="col-xs-1 form-group">' +
+                           '<button class="btn btn-info btn-remove form-control" type="button"><span class="glyphicon glyphicon-minus"></span></button>' +
+                         '</div>' +
+                       '</div>');
+
+      addPlateOptions(newEntry.find('.plate-select:first'));
+      newEntry.appendTo(controlForm);
+      initGUI();
+    }).on('click', '.btn-remove', function(e) {
+      $(this).parents('.entry:first').remove();
+      e.preventDefault();
+      initGUI();
+      return false;
+    });
+  });
+</script>
+{% end %}
+{% block content %}
+<h3>Prepare targeted pool</h3>
+<div class="container">
```

Comment:
It looks to me that some HTML codes are duplicated? e.g., `<div class="entry row">`